scholarx/past-programs/2020/styles.css

```diff
@@ -79,6 +79,14 @@
     padding: 1.25rem;
 }
 
+.program-timeline-row{
+    width: 100%;
+}
+
+.program-timeline-date{
+    text-align: end !important;
+}
+
```

Comment:
Let's move this to` /assets/css/global-custom.css` so we can avoid duplicating the same code